File: Arrays/Array math/Min Steps in Infinite Grid.java
public class Solution {
    public int coverPoints(ArrayList<Integer> x, ArrayList<Integer> y) {
        int ans=0;
        for(int i=0;i<x.size()-1;i++){
            int xdiff = Math.abs(x.get(i)-x.get(i+1));
            int ydiff = Math.abs(y.get(i)-y.get(i+1));
            ans += Math.max(xdiff,ydiff);
        }
        return ans;
    }
}

File: Arrays/Array math/Min Steps in Infinite Grid.py
class Solution:
	# @param x : list of integers
	# @param y : list of integers
	# @return an integer
	def coverPoints(self, x, y):
        ans=0
        for i in range(len(x)-1):
            xdiff = abs(x[i]-x[i+1]);
            ydiff = abs(y[i]-y[i+1]);
            ans += max(xdiff,ydiff);
        return ans;


